generators/spring-service/index.js

```diff
@@ -23,14 +23,21 @@ const constants = require('../generator-constants');
 
 const SERVER_MAIN_SRC_DIR = constants.SERVER_MAIN_SRC_DIR;
 
+let useBlueprint = false;
+
 module.exports = class extends BaseGenerator {
     constructor(args, opts) {
         super(args, opts);
-        this.argument('name', { type: String, required: true });
```

Comment:
why is the argument removed? without it the help will not show it I guess

Reply:
`this.argument` is not having name in it.

Since, there are two arguments here.

Reply:
I dont get it, service name is the only argument rite? can you explain the problem? this is working code right

Reply:
w.r.t blueprint this is coming empty

Reply:
it should work, if not there might be something wrong. Needs to be checked. We cant remove the argument declaration as it would break validation and discovery in help provided by yeoman

Reply:
Yeah, that is the main reason why I had put `don't merge` for the PR.

Let me describe the problem now,

* When you run `jhipster --blueprint generator-jhipster-kotlin spring-service foo`
* The arguments inside the blueprint's generator is empty and the blueprint generator is calling the main generator, hence there also the arguments are empty instead the argument is present in the `_` property of the object.

Is this the expected behavior or do we have to do something different to make sure that the arguments are sent across for `yeoman` generator?